api/krusty/managedbylabel_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	kusttest_test "sigs.k8s.io/kustomize/api/testutils/kusttest"
 )
 
-func TestAddManagedbyLabel(t *testing.T) {
```

Comment:
suggest using the parameterized tests for  TestAddManagedbyLabelFromFlag and TestAddManagedbyLabelFromField.

Reply:
done!